src/libraries/System.Text.Encodings.Web/src/System/Text/Encodings/Web/DefaultJavaScriptEncoderBasicLatin.cs

```diff
@@ -304,17 +325,19 @@ public override unsafe int FindFirstCharacterToEncodeUtf8(ReadOnlySpan<byte> utf
                     Debug.Assert(pValue <= vectorizedEnd && vectorizedEnd <= (pValue + utf8Text.Length - Vector128<byte>.Count));
 
                     // Load the last 16 bytes
-                    Vector128<sbyte> sourceValue = Sse2.LoadVector128((sbyte*)vectorizedEnd);
+                    Vector128<sbyte> sourceValue = Sse2.IsSupported ?
+                        Sse2.LoadVector128((sbyte*)vectorizedEnd) :
+                        AdvSimd.LoadVector128((sbyte*)vectorizedEnd);
```

Comment:
>AdvSimd.LoadVector128((sbyte*)vectorizedEnd); [](start = 24, length = 45)

Just have a dedicated check for `AdvSimd.Arm64.IsSupported` here. It is replaced by `true` in JIT on ARM64.

Reply:
`AdvSimd.Arm64.IsSupported` is a shimmed constant in `netcoreapp3.0`, so checking for that property would result in compile-time warnings.